src/MICore/MICommandFactory.cs

```diff
@@ -501,6 +501,34 @@ public virtual void DecodeExceptionReceivedProperties(Results miExceptionResult,
 
         #region Helpers
 
+        /// <summary>
+        /// Generates results that represent an emulated MI stopped record.
+        /// </summary>
+        public async Task<Results> GenerateStoppedRecordResults()
```

Comment:
Is the command factory the right place for this code to live? Isn't command factory normally used to send commands to the debuggee?